.evergreen/.evg.yml

```diff
@@ -119,8 +122,9 @@ functions:
           find $MONGO_ORCHESTRATION_HOME -name \*.log | xargs tar czf mongodb-logs.tar.gz
     - command: s3.put
       params:
-        aws_key: ${aws_key}
-        aws_secret: ${aws_secret}
+        aws_key: ${AWS_ACCESS_KEY_ID}
```

Comment:
There are the same properties that we use for CSFLE/QE testing.  I'm surprised to see them used here as well.  Is this correct?

Reply:
It seems correct but suspicious. Since the [ec2.assume_role](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Commands#ec2assume_role) command is invoked directly above, and we do not use those project variables after, they should not override what we have set. I can confirm that the upload succeeds in the logs once dev prod provides us with the new `ASSUME_ROLE_ARN` value.

However, it is unclear why we have both `aws_access_key_id` and `aws_key` set in our project config (they have different values), and I would imagine that aws_access_key_id should be treated as reserved. We also have various keys that have similar values, such as `iam_auth_assume_aws_account` and `iam_auth_ecs_account`. Should these be changed to assume roles as well? [Other drivers](https://github.com/mongodb/mongo-csharp-driver/blob/27ee0b4c5c59737f3f2efc85bfea65f795f7ff25/evergreen/evergreen.yml#L134) use static credentials, but also use `ec2.assume_role` in other places.

My inclination is to merge this in, to get things working for the new project/branch, and then do any other work under a new DRIVERS ticket created by dev prod (it should be possible to identify static keys to be deprecated, for each driver, and get everyone switched over in a scheduled way; this would save the issue from arising during something like a release).

Reply:
> However, it is unclear why we have both aws_access_key_id and aws_key set in our project config (they have different values),

* `aws_key` is in pretty much every Evergreen project config and its value is unique to each project. It's generally used for pushing artifacts (e.g log files, test results) to S3.
* `aws_access_key_id` is specifically for CSFLE/QE drivers testing and its value is the same for all drivers.  While it seems possible to use it to push to S3, that's not what it's intended for.